cypress/integration/e2e/article.interactivity.spec.js

```diff
@@ -31,6 +33,7 @@ describe('Interactivity', function() {
 
         it('should display all the rich links for an article', function() {
             cy.visit(`/Article?url=${articleUrl}`);
+            cy.scrollTo('bottom', { duration: 100 });
```

Comment:
Will you remove all this now?

Reply:
No, I had to solve the problem _both_ for Cypress **and** for Chromatic

Reply:
Ah yeah course